code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -393,6 +393,16 @@
 	new /obj/item/card/id/prisoner/random(src)
 	new /obj/item/radio/headset(src)
 
+/obj/structure/closet/secure_closet/vr
+	name = "VR exile"
+	req_access = list(access_security)
+	anchored = 1
+
+/obj/structure/closet/secure_closet/vr/New()
+	..()
+	for(var/i in 0 to 7)

```

Comment:
Weird that this does `0 to 7` as opposed to `1 to 8` 😄

Reply:
I'm used to counting from zero damnit